README.md

```diff
@@ -18,42 +18,34 @@ Then, select one of the following methods.
 
 ### Using Carthage
 
-Checkout the tag PolyNetSDK-version-5-carthage in the repository.
-
-```shell
-git checkout PolyNetSDK-version-5-carthage
-```
-
-Update the carthage dependencies for the specific Apple platform. For example, iOS for PolyNetSampleSwift.
+Update the carthage dependencies for the specific Apple platform. 
 
 ```shell
 carthage update --platform ios --use-xcframeworks
 ```
 
-Run the project from Xcode.
+Go to the Frameworks folder inside the Xcode project (.xcodeproj) and remove the PolyNetSDK, Starscream, and SwiftProtobuf frameworks that are marked in red and which not being found by Xcode. After removing them, right-click inside the frameworks folder and select Add files to PolyNetSampleSwift. Make sure copy items if needed, create groups and add to targets PolyNetSampleSwift is selected. Then choose the XCFrameworks (PolyNetSDK, Starscream and SwiftProtobuf) from the PolyNetSampleSwift/Carthage/Build folder. 
+
+Once added, run the project from Xcode.
 
 ### Using CocoaPods
 
-Checkout the tag PolyNetSDK-version-5-cocoapods in the repository.
+Install or Update the dependencies.
 
 ```shell
-git checkout PolyNetSDK-version-5-cocoapods
+pod update
 ```
 
-Install the dependencies.
-
-```shell
-pod install
-```
+*Beware to open the workspace (.xcworkspace) and not the project (.xcodeproj) itself to work with CocoaPods.*
 
-Run the created workspace from Xcode.
+Go to the Frameworks folder inside the Xcode project workspace (.xcworkspace) and delete the PolyNetSDK, Starscream, and SwiftProtobuf frameworks that are marked in red and which not being found by Xcode. Only keep the Pods_PolyNetSampleSwift.framework. 
```

Comment:
Open the .xcworkspace file of your Xcode project and ensure that only Pods_PolyNetSampleSwift.framework is present in the Frameworks, Libraries, and Embedded Content sections.

Reply:
Done